tests/test-lxd-rbac

```diff
@@ -0,0 +1,189 @@
+#!/bin/sh
```

Comment:
Can you rename the file to drop the `test-lxd-` prefix? I'll be renaming the other scripts in another PR :)